composer.json

```diff
@@ -50,7 +51,7 @@
         "barryvdh/laravel-ide-helper": "2.10.0",
         "filp/whoops": "2.14.4",
         "friendsofphp/php-cs-fixer": "3.2.1",
-        "fzaninotto/faker": "1.9.2",
```

Comment:
can this stay at 1.9.2?

Reply:
I have changed it because when I tried to install the `spatie/array-to-xml` bundle composer returned a error. And If you check the composer.json for the fzaninotto/faker package v1.9.2 the php require is: `"^5.3.3 || ^7.0"`, so, composer can return error for php 8.

https://github.com/fzaninotto/Faker/blob/v1.9.2/composer.json#L17

Reply:
ok, could you update faker to refer to an actual version and not a range? you can get the installed version from composer.lock